### ISSUE TITLE
`--showConfig` is ignoring missing `extends` file

### ISSUE THREAD

**Issue by vvakame** [Microsoft/TypeScript#28598]
**TypeScript Version:**  3.2.0-dev.20181117

and local build with 608a8307accdfdf0f0ae8fb8b356adedee939d09

**Search Terms:**
showConfig

**Code**

```tsconfig.json
{
  "extends": "./notExists.tsconfig.json",
  "compilerOptions": {
    "target": "esnext",
    "module": "commonjs",
    "strict": true
  }
}
```

**Expected behavior:**
show `error TS5058: The specified path does not exist: '/foobar/notExists.tsconfig.json'.` error message.

**Actual behavior:**
don't report any error and dump incompleteness config value.

**Playground Link:** None

**Related Issues:** #27353

I made a patch to this issue.
If this is a problem, I can submit this patch.
https://github.com/vvakame/TypeScript/commit/d6d9c9ec2053080f4ac632357a422e28e03f7391

**Comment by weswigham** [comment]
I've gone back and forth on this - the diagnostics are obviously useful for fixing the config; however then you can't rely on the output with the flag on always being JSON. Then again, the return code changes when diagnostics are emitted, so that might be fine...

@vvakame I think we'll take a PR for the change, but the JSON document and the diagnostics should be either/or - meaning if there are diagnostics for the config file, we print those, rather than the config. I think that's probably the way to go.
